bin/startPipeline.sh

```diff
@@ -172,23 +172,54 @@ function submitPipeline () {
 		log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "Skipping  ${_project}/${_run}, because jobs have already finished."
 		return
 	fi
-	
+
 	if [[ ! -e "${TMP_ROOT_DIR}/logs/${_project}" ]]
 	then
 		mkdir -p "${TMP_ROOT_DIR}/logs/${_project}"
 	fi
-	
+
 	cd "${TMP_ROOT_DIR}/projects/${_project}/${_run}/jobs/"
 	log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' "Navigated to: ${TMP_ROOT_DIR}/projects/${_project}/${_run}/jobs/"
-	
+
+	declare -a sampleSheetColumnNames=()
+        declare -A sampleSheetColumnOffsets=()
```

Comment:
Messy indentation in entire file.
Please leave blank lines idented, which is required for some editors to allow increasing or decreasing the indentation level for code blocks.